tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -49,7 +50,7 @@
 from tests_e2e.tests.lib.agent_test import TestSkipped
 from tests_e2e.tests.lib.agent_test_context import AgentTestContext
 from tests_e2e.tests.lib.identifiers import VmIdentifier
-from tests_e2e.tests.lib.logging import log as agent_test_logger  # Logger used by the tests
```

Comment:
Initally most of the output was going to the LISA log, which was referenced as 'self.log', while the the test logs were referenced as 'agent_test_logger'.

Over time, most of the output has been moved to the test logs. Now the LISA log is references as 'self.lisa_log' and the test logs simply as 'log' (which makes this class consistent with the naming in the rest of the test code).